### ISSUE TITLE
Banks do not switch when clicking A B C D buttons

### ISSUE THREAD

**Issue by ViktorNova** [harryhaaren/openAV-Fabla2#34]
On the latest build, clicking the A B C D buttons has no affect, except for changing the colors of various UI elements. Watching the output, it appears to stay on Bank 0 regardless of which bank is chosen

**Comment by harryhaaren** [comment]
Yes - this is a decision to try release 2.0 sooner - one bank. The code for 4 banks remains, just untested and hence de-activated. I'll add to the 2.1 milestone.

**Comment by ViktorNova** [comment]
AHHH, this makes sense! 
Thank you for clarifying. I would be happy to help test the banks function if you ever need help =)
